serial/serial_resources/process_entry.py

```diff
@@ -0,0 +1,149 @@
+import serial
+import os
+import mimetypes
+from datetime import datetime
+from ..api_client import APIClient 
+from .dataset import Datasets 
+
+class ProcessEntry:
+    def __init__(self, process_entry_data): 
+        """
+        Creates a new Process Entry
+        
+        Args:
+        - client: A Serial client object
+        - component_instance: Component Instance object 
+        - process_id: UUID for the process associated with this entry
+
+        Returns:
+        - A newly created process entry
+        """
+        self.client = APIClient(serial.api_key, serial.base_url) 
+        self.data = process_entry_data
+        self.process_id = process_entry_data["process_id"] 
+        self.id = process_entry_data["id"] 
+
+    def add_text(self, dataset_name, value, expected_value=None):
+        """
+        Add text data to a process entry
+
+        Args: 
+        - dataset_name: User facing name of the dataset
+        - value: The value to be submitted
+        - expected_value: Optional argument to pass an expected value. If the 
+        values are the same, it would be considered a passing test result
+        """
+        if serial.debug:
+            print(f"Adding text data: {dataset_name} with value: {value} and expected value: {expected_value}")
+        dataset = serial.Datasets.get(dataset_name, "PARAMETRIC_QUALITATIVE", self.process_id)
```

Comment:
The get method refers to `GET /datasets` which should take a the new types not the old ones right? Ie. this string should be "TEXT". I think datasets.py might have to be tweaked too (`type` not `data_type`)?
![image](https://github.com/serialmfg/serial-py/assets/8787035/e271c39c-3647-4228-bcd4-a32a2e58ac93)

Reply:
This has been updated in the documentation